frontend/dance-with-me/src/components/Landing/Landing.jsx

```diff
@@ -55,7 +54,7 @@ function Landing() {
     />
 
     <Carousel.Caption>
-      <h3><Login2/></h3>
+      <h3><Login text="Find your matches" color="primary"/></h3>
```

Comment:
Hi Uufuk, nice job well done. I have small feedbzck here; "Be" word is not visible beacuse of picture color. We may use span inline <p> element. 
<p class="pl-5 text-uppercase font-weight-bold"><span class="text-light">Be </span ><span class="text-dark">active together with DanceWithMe</span></p>
like this as well as we may add lower opacity bg-clor what do you think?